docs/en/changes/changes.md

```diff
@@ -97,6 +97,8 @@
   * Support different indexType
   * Support configuration for TTL and (block|segment) intervals
 * Optimize MQ Topology analysis. Use entry span's peer from the consumer side as source service when no producer instrumentation(no cross-process reference). 
+* Elasticsearch storage: Provide system environment variable(`SW_STORAGE_ES_SPECIFIC_INDEX_SETTINGS`) and support specify the settings `(number_of_shards/number_of_replicas)` for each index individually.
+* Elasticsearch storage: Support update index settings `(number_of_shards/number_of_replicas)` for the index template after re-configured.
```

Comment:
```suggestion
* Elasticsearch storage: Support update index settings `(number_of_shards/number_of_replicas)` for the index template after rebooting.
```

I think we need to make this and doc consistent. The doc doesn't mention rebooting and override existing index settings.